File: src/main/java/com/bajdcc/LALR1/interpret/os/ui/UIHitokoto.java
package com.bajdcc.LALR1.interpret.os.ui;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【界面】一言
 *
 * @author bajdcc
 */
public class UIHitokoto implements IOSCodePage {
	@Override
	public String getName() {
		return "/ui/hitokoto";
	}

	@Override
	public String getCode() {
		return ResourceLoader.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/ui/UIHitokoto.kt
package com.bajdcc.LALR1.interpret.os.ui

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【界面】一言
 *
 * @author bajdcc
 */
class UIHitokoto : IOSCodePage {
    override val name: String
        get() = "/ui/hitokoto"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
